sdk/src/tx/baseTxSender.ts

```diff
@@ -25,12 +25,13 @@ import assert from 'assert';
 import bs58 from 'bs58';
 import { TxHandler } from './txHandler';
 import { IWallet } from '../types';
+import { BankrunConnection } from '../bankrunConnection';
 
 const DEFAULT_TIMEOUT = 35000;
 const NOT_CONFIRMED_ERROR_CODE = -1001;
 
 export abstract class BaseTxSender implements TxSender {
-	connection: Connection;
+	connection: Connection | BankrunConnection;
```

Comment:
think everywhere we had to add union types in sdk, would be nicer to just do ts-ignore in sdk

Reply:
might also be able to somehow hack it into TestClient as well